### PR TITLE
Update to latest NuGet.Services.Sql

### PR DESCRIPTION
**Pull request by chenriksson**
Along with the following PRs, would update Gallery to use SQL AAD authentication... will update to master build before merge. DEV only for now.
- https://github.com/NuGet/ServerCommon/pull/172
- [NuGetDeployment#516](https://nuget.visualstudio.com/NuGetMicrosoft/_git/NuGetDeployment/pullrequest/516?_a=overview)

Planning DEV deployment today to verify changes before merging.

### CODE REVIEW

File: src/NuGetGallery.Core/NuGetGallery.Core.csproj [hunk 1]
```diff
@@ -246,10 +246,10 @@
       <Version>4.8.0-preview4.5287</Version>
     </PackageReference>
     <PackageReference Include="NuGet.Services.Validation">
-      <Version>2.26.0-master-33196</Version>
+      <Version>2.27.0-master-35322</Version>
     </PackageReference>
     <PackageReference Include="NuGet.Services.Validation.Issues">
-      <Version>2.26.0-master-33196</Version>
+      <Version>2.27.0-master-35322</Version>
```
Comment (joelverhagen) [inline src/NuGetGallery.Core/NuGetGallery.Core.csproj]: Can we have a stable build of ServerCommon?

File: src/NuGetGallery/Diagnostics/TraceDiagnosticsSource.cs [hunk 2]
```diff
@@ -94,5 +95,38 @@ private static string FormatMessage(string message, string member, string file,
...
+        {
+            return true;
+        }
+
+        IDisposable ILogger.BeginScope<TState>(TState state)
+        {
+            return null;
+        }
+
+        private static TraceEventType LogLevelToTraceEventType(LogLevel logLevel)
```
Comment (joelverhagen) [inline src/NuGetGallery/Diagnostics/TraceDiagnosticsSource.cs]: Could we use this as a model?
https://github.com/aspnet/Logging/blob/d9b34d1c36fde204bd736d09b89b86451bd82e95/src/Microsoft.Extensions.Logging.TraceSource/TraceSourceLogger.cs#L58-L69

Namely, `Warning` is missing.

File: src/NuGetGallery/Diagnostics/TraceDiagnosticsSource.cs [hunk 3]
```diff
@@ -94,5 +95,38 @@ private static string FormatMessage(string message, string member, string file,
...
+            TraceEvent(LogLevelToTraceEventType(logLevel), eventId.Id, message);
+        }
+
+        bool ILogger.IsEnabled(LogLevel logLevel)
+        {
+            return true;
+        }
+
+        IDisposable ILogger.BeginScope<TState>(TState state)
+        {
```
Comment (joelverhagen) [inline src/NuGetGallery/Diagnostics/TraceDiagnosticsSource.cs]: Is it in contract for this to return null?

File: src/NuGetGallery.Core/NuGetGallery.Core.csproj (hunk 1 above)
Comment (chenriksson) [reply src/NuGetGallery.Core/NuGetGallery.Core.csproj]: Will do before merge

File: src/NuGetGallery/Diagnostics/TraceDiagnosticsSource.cs (hunk 2 above)
Comment (chenriksson) [reply src/NuGetGallery/Diagnostics/TraceDiagnosticsSource.cs]: Updated

File: src/NuGetGallery/Diagnostics/TraceDiagnosticsSource.cs (hunk 3 above)
Comment (chenriksson) [reply src/NuGetGallery/Diagnostics/TraceDiagnosticsSource.cs]: Updated per aspnet/Logging example you pointed me to - many thanks!

**Pull request merged**